examples/keras_recipes/melanoma_classification_with_tfrecords.py

```diff
@@ -0,0 +1,336 @@
+"""
+Title: How to train a Keras model on TFRecord files
+Author: Amy MiHyun Jang
+Date created: 2020/07/29
+Last modified: 2020/08/06
+Description: Loading TFRecords for computer vision models
+"""
+"""
+## Introduction + Set Up
+
+TFRecords store a sequence of binary records, read linearly. They are useful format for
+storing data because they can be read efficiently. We'll explore how we can easily load
+in TFRecords for our melanoma classifier.
+"""
+
+"""shell
+! pip install gcsfs -q
+"""
+
+import re
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from functools import partial
+import matplotlib.pyplot as plt
+
+try:
```

Comment:
I recommend remove all TPU-related code, as this is outside the scope of TFRecord loading